extensions/amp-experiment/1.0/mutation-parser.js

```diff
@@ -28,6 +30,34 @@ const MUTATION_TYPES = [
   'childList',
 ];
 
+const SUPPORTED_ATTRIBUTES = {
+  style: value => {
+
+    // Do not allow Important or HTML Comments
+    if (value.match(/(!\s*important|<!--)/g)) {
+      return false;
+    }
+
+    // Allow Color
+    if (value.match(/^color:\s#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/g)) {
+      return true;
+    }
+
+    // Allow Background color
+    if (value.match(/^background-color:\s#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/g)) {
```

Comment:
Im fine with this for now, but I feel like eventually the property validation should be separated from the value validation.

Reply:
Totally agree, after our demo, we will probably be moving this to the Validator. If not, I added an note here: https://github.com/ampproject/amphtml/issues/21705 😄

Reply:
nit/bikeshedding:

Why did you go with just `\s` and not `\s*` after the colon (`:`)?

And why no whitespace before the colon or before/after the entire value set?

not nit-picking:

Why the `/g`? shouldn't there be only one `background-color`? what does the global accomplish or am I misunderstanding it (happens with regexs)

😁